lib/code_corps/stripe/in_memory.ex

```diff
@@ -0,0 +1,24 @@
+defmodule CodeCorps.Stripe.InMemory do
+  defmodule Customer do
+    def create(map) do
+      {:ok, do_create(map)}
+    end
+
+    defp do_create(_) do
+      {:ok, created} = DateTime.from_unix(1479472835)
+
+      %Stripe.Customer{
+        id: "cus_9aMOFmqy1esIRE",
+        account_balance: 0,
+        created: created,
+        currency: "usd",
+        default_source: nil,
+        delinquent: false,
+        description: nil,
+        email: "mail@test.com",
+        livemode: false,
+        metadata: %{}
+      }
+    end
```

Comment:
Further, now that I look at the effort we're putting into this, I'm not convinced that this shouldn't be happening in `stripity-stripe` or a related testing library, lest we reproduce the whole surface area of that library in our mocked API.